spec/models/solidus_pay_tomorrow/payment_source_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe SolidusPayTomorrow::PaymentSource, type: :model do
+  let(:payment_method) { create(:pt_payment_method) }
+  let(:payment_source) { build(:pt_payment_source, payment_method: payment_method) }
+
+  describe 'validations' do
+    context 'when payment_method_id present' do
+      it 'is valid' do
+        expect(payment_source.valid?).to be(true)
+      end
+    end
+
+    context 'when payment_method_id absent' do
+      before { payment_source.payment_method_id = nil }
+
+      it 'is invalid' do
+        expect(payment_source.valid?).to be(false)
+        expect(payment_source.errors.messages.first.to_a).to eq([:payment_method_id, ["can't be blank"]])
```

Comment:
What do you think about adding shoulda_matcher and using it for these tests?

Reply:
Can do

Reply:
@DanielePalombo Let me know if this needs to be done. I think we'd add `shoulda_matchers` if we need them in future also

Reply:
Yes, I think it is worth it if we have other validation in the future.
Adding shoulda_matcher with `add_development_dependency ` doesn't require it to the application environment.

Reply:
Cool. Added in https://github.com/nebulab/solidus_pay_tomorrow/pull/12/commits/70b8f06946ce127597144fcb78db0e365c2ae40b
and updated this spec